Documentation editorial review (#22185)
DataDog/documentation: content/en/metrics/volume.md

The Volume page displays a list of your metrics reported to Datadog sorted by estimated custom metrics or by the change in volume. To sort metrics by either of these options, click on the column header of the metric table.
| Column | Description |
|--------|-------------|
|**Top 500 Metric Names by Estimated Real-time Cardinality** | Identify the top 500 metric names by cardinality/volume.| 
|**Top 500 Metric Names by Change in Volume** |Discover the top 500 metric names that have the greatest variance in their cardinality. These metrics may have anomalously (potentially unintentionally) spiked in the timeframe of your choosing. If you receive an alert on your account's estimated real-time custom metrics usage, you can use this view to investigate the metric spike. |

## Compare metric cardinality

{{< img src="metrics/volume/compare_metric_cardinality.png" alt="Metrics Volume filtered down to metric names with “shopist”, sorted by estimated custom metrics. On hover over the change in volume, displays the cardinality graph of the metric over the past day" style="width:100%;" >}}

> Cardinality refers to the number of unique sets of tag values associated with a given key. Tags with very large numbers of possible values have a "high cardinality". A custom metric is identified by a unique combination of a metric's name and tag values (including the host tag). 

Author: ```suggestion
Cardinality refers to the number of unique sets of tag values associated with a given key. Tags with large numbers of possible values have a "high cardinality". A custom metric is identified by a unique combination of a metric's name and tag values (including the host tag). 
```

Compare metric cardinality to understand:
  - Which metric names are causing your custom metrics bill to spike. 
  - What the organization's largest custom metrics are.
  - What metric names have spiked recently in the past day or month.

To view your spiking metric's cardinality over time:

> 1. Select a timeframe in the top right hand corner (the recommended timeframe is **Past 1 Day** or **Past 4 Weeks**).

Author: ```suggestion
1. Select a time frame in the top right hand corner (the recommended time frame is **Past 1 Day** or **Past 4 Weeks**).
```

To effectively reduce costs with Metrics without Limits, identify your highest volume or spiking metric names. To find the metrics not actively queried in the past 30 days, click on **Not Actively Queried** in the Query Activity Facet box. Datadog analyzes query patterns to determine valuable metrics. Selecting **Not Actively Queried** generates a list of unused metric names across dashboards, notebooks, monitors, SLOs, Metrics Explorer, and the API.

## Reduce metric volume and cost

After you identify unqueried metrics, you can eliminate the volume and cost of these metric names by using [Metrics without Limits™][3]:

1. Click on the metric name to open its details side panel.
2. Click the **Manage Tags** button to open the tag configuration modal.
3. Select **Include tags…** and set an empty allowlist of tags.

You can also click the **Configure Metrics** dropdown and select **Manage Tags** to open the [Metrics without Limits™ Tag configuration modal][4]. 

{{< img src="metrics/volume/configure_metrics.png" alt="Configure Metric dropdown at the top of the page highlighting the Manage tags option" style="width:100%;" >}}

> You have full control over the cardinality of your metrics at any time with zero changes needed to your applications or the requirement of a remote-write setup. Below is an example of how eliminating timeseries that are never or rarely leveraged can significantly reduce your custom metrics volumes and therefore costs.

Author: ```suggestion
You have full control over the cardinality of your metrics without the need to change your applications nor the requirement of a remote-write setup. Below is an example of how eliminating timeseries that are rarely used can significantly reduce your custom metrics volumes and costs.
```
